static/sass/_charmhub_footer.scss

```diff
@@ -8,9 +8,23 @@
         rgba(119, 41, 83, 0) 50%,
         rgba(119, 41, 83, 0) 100%
       ),
-      linear-gradient(266deg, #044f66 99%, #022935);
-    background-size: 25% 15%, 100%;
-    background-position: top right;
+      linear-gradient(
+        to top right,
+        rgba(216, 216, 216, 0.04) 0%,
+        rgba(216, 216, 216, 0.04) 49.9%,
+        rgba(216, 216, 216, 0) 50%,
+        rgba(216, 216, 216, 0) 100%
+      ),
+      linear-gradient(
+        to bottom right,
+        rgba(216, 216, 216, 0.04) 0%,
+        rgba(216, 216, 216, 0.04) 49.9%,
+        rgba(216, 216, 216, 0) 50%,
+        rgba(216, 216, 216, 0) 100%
+      ),
+      linear-gradient(266deg, #044f66 100%, #022935);
+    background-size: 25% 15%, 60% 100%, 25% 100%, 100%;
+    background-position: top right, bottom left, bottom left;
```

Comment:
Try: 
```scss
$brand-color: #044f66;
background-blend-mode: overlay, overlay, normal;
background-color: $brand-color;
background-image:
  linear-gradient(to bottom left, rgba(216, 216, 216, 0.15) 0, rgba(216, 216, 216, 0.15) 49.9%, transparent 50%),
  linear-gradient(to bottom right, transparent 50%, rgba(216, 216, 216, 0.15) 0, rgba(216, 216, 216, 0.15) 49.9%),
  linear-gradient(-89deg, $brand-color 0%, darken($brand-color, 10%) 94%);
background-position: top right, top right, 0% 0%;
background-repeat: no-repeat;
background-size: 50% 100%, 15% 100%, auto;
color: $color-x-light;
```

I would move the brand color to a settings file like in vanilla.

Reply:
@anthonydillon yes, I will do that, however the general problem we have is the 3rd linear gradient which looks like this `linear-gradient(268deg, #044f66 100%, #022733);` on zeppelin and because it has the first colour set  to `100%`, the other colour is not visible. But we will discuss more in the meeting later.